python/paddle/v2/dataset/common.py

```diff
@@ -32,3 +32,10 @@ def download(url, module_name, md5sum):
             shutil.copyfileobj(r.raw, f)
 
     return filename
+
+
+def dict_add(a_dict, ele):
```

Comment:
Does `dict_add` need to be added `__all__` before other paddle paddle packages being able to use it?